lib/modules/tests/test.js

```diff
@@ -0,0 +1,326 @@
+const async = require('async');
+const AccountParser = require('../../utils/accountParser');
+const EmbarkJS = require('embarkjs');
+const utils = require('../../utils/utils');
+
+class Test {
+  constructor(options) {
+    this.options = options || {};
+    this.simOptions = {};
+    this.events = options.events;
+    this.logger = options.logger;
+    this.ipc = options.ipc;
+    this.configObj = options.config;
+    this.ready = true;
+    this.firstRunConfig = true;
+    this.error = false;
+    this.contracts = {};
+    this.firstDeployment = true;
+    this.needConfig = true;
+    this.provider = null;
+    this.accounts = [];
+  }
+
+  init(callback) {
+    this.gasLimit = 6000000;
```

Comment:
Same here. This arbitrary gasLimit probably better put in a constant outside of this class. Ideally we have this somewhere defined and exposed, so we can import it where needed...